internal/signer/darwin/signer.go

```diff
@@ -32,17 +32,34 @@ import (
 	"github.com/googleapis/enterprise-certificate-proxy/internal/signer/util"
 )
 
+
 // If ECP Logging is enabled return true
 // Otherwise return false
-func enableECPLogging() bool {
+func enabledECPLogging() bool {
 	if os.Getenv("ENABLE_ENTERPRISE_CERTIFICATE_LOGS") != "" {
 		return true
 	}
-
-	log.SetOutput(io.Discard)
 	return false
 }
 
+func ecpLogf(format string, v ...any) {
```

Comment:
I was thinking instead of creating 1:1 wrappers for the existing log statements, make a more user-friendly logging util, which also incorporates the "INFO/DEBUG/WARN" conventions. You can use the terraform logging env var mechanism for reference, which I believe is well-implemented. Let me know if you have questions. Thanks!